python/requirements/ml/requirements_train.txt

```diff
@@ -12,6 +12,7 @@ tensorboardX==2.4.1
 # `python/ray/air/tests/test_huggingface_trainer.py`
 transformers==4.18.0; python_version <= '3.6'
 transformers==4.19.1; python_version > '3.6'
-accelerate==0.5.1
+accelerate==0.5.1; python_version <= '3.6'
+accelerate==0.17.1; python_version > '3.6'
```

Comment:
Can you make sure we document that this is what we test against? You can just list the 0.17.1 version.

Reply:
where should we document that? here, in the test file, or in the docs?

Reply:
docstring ?